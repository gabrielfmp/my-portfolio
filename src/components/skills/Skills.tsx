import {
  AlignVertically,
  SkillText,
  SkillsLine,
  SkillsWrapper,
} from './styles';
import React from 'react';

const Skills = () => {
  return (
    <SkillsWrapper>
      <AlignVertically>
        <SkillsLine>
          <SkillText>Critical Thinking</SkillText>
          <SkillText>Will to learn</SkillText>
        </SkillsLine>

        <SkillsLine>
          <SkillText>English</SkillText>
          <SkillText>Portuguese (native)</SkillText>
          <SkillText>Spanish</SkillText>
        </SkillsLine>

        <SkillsLine>
          <SkillText>Communication</SkillText>
          <SkillText>Self-improvement</SkillText>
        </SkillsLine>

        <SkillsLine>
          <SkillText>Creativity</SkillText>
          <SkillText>Organization</SkillText>
          <SkillText>Teamwork</SkillText>
        </SkillsLine>
      </AlignVertically>
    </SkillsWrapper>
  );
};

export default Skills;
