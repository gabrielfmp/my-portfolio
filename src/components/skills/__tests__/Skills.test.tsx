import { render } from '@testing-library/react';
import React from 'react';
import Skills from '../Skills';

test('snapshot', () => {
  const { asFragment } = render(<Skills />);

  expect(asFragment()).toMatchSnapshot();
});
