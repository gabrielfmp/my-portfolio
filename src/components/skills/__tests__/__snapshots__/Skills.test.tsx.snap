// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-1vymck9"
  >
    <div
      class="css-o7xbd4"
    >
      <div
        class="css-1sulisw"
      >
        <p
          class="css-1yo0kyw"
        >
          Critical Thinking
        </p>
        <p
          class="css-1yo0kyw"
        >
          Will to learn
        </p>
      </div>
      <div
        class="css-1sulisw"
      >
        <p
          class="css-1yo0kyw"
        >
          English
        </p>
        <p
          class="css-1yo0kyw"
        >
          Portuguese (native)
        </p>
        <p
          class="css-1yo0kyw"
        >
          Spanish
        </p>
      </div>
      <div
        class="css-1sulisw"
      >
        <p
          class="css-1yo0kyw"
        >
          Communication
        </p>
        <p
          class="css-1yo0kyw"
        >
          Self-improvement
        </p>
      </div>
      <div
        class="css-1sulisw"
      >
        <p
          class="css-1yo0kyw"
        >
          Creativity
        </p>
        <p
          class="css-1yo0kyw"
        >
          Organization
        </p>
        <p
          class="css-1yo0kyw"
        >
          Teamwork
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
